### DISCUSSION TITLE
Priority sorting; "no priority" location

### DISCUSSION THREAD

**Discussion by AeliusSaionji** [TechbeeAT/jtxBoard#652] [General]
Hi!
I wanted to ask if others prefer "no priority" to be at the top of the list when sorting/grouping by priority.

To me, it seems odd that this ranks higher than "highest", but maybe I'm overlooking some useful workflow or setting. Interested to get others' perspective on this.

Thanks!

**Comment by patrickunterwegs** [comment]
Hey @AeliusSaionji , I just checked this, the "no priority" is actually always at the bottom of the list, no matter if you sort ascending or descending or if you use the group by option. Are you sure you have set the primary sort order to priority in your case?

**Reply by AeliusSaionji** [reply]
![Screenshot_20230624-165324](https://github.com/TechbeeAT/jtxBoard/assets/4342746/3bc7aba4-36c7-4c9d-99e4-4487697b015a)


Is this a bug then?

**Reply by patrickunterwegs** [reply]
Which version are you using?

**Reply by AeliusSaionji** [reply]
2.04.07.ose

**Reply by patrickunterwegs** [reply]
It's very weird now, on different emulators it just works properly (showing always entries without a priority last), but on a real device the behaviour is different... I need to do some more investigation here...
